### ISSUE TITLE
Locale set to USA causes no E131 data sent to certain channels 

### ISSUE THREAD

**Issue by jimtxas** [FalconChristmas/fpp#924]
FPP v4.5.1-7 - Pi4

As bizarre as this is going to sound, i upgraded to 4.5.x recently and begun having a group of pixels not responding on universe  122-130.  I do not know the significance of these universes but channels affected were roughly: 21421 - 26010.

Aside from visually seeing that models were not lighting up as should, i realized i was able to see this in the F48 status console as well - noted in the screen shots.  

What i found odd is if i ran an output test from FPP i could get data on those channels if i chose all channels.  If i chose just the models, nothing.  What appeared to be happening is those models were receiving data on channels at the end of that universe spectrum but not against the channels noted above.  Again, switching this back to global made this issue disappear.

Ultimately after much trial and error, several re-images, etc i was able to isolate the issue to the locale setting to USA in FPP.  When browsing the latest settings i was changing this to USA because, well i'm located in the USA.  The details related to this setting to not suggest why this was happening but i can confirm that setting this back to global corrects the issue and those affected universes and channels begin receiving data when NOT set to USA.  Hours of looking at F48 settings, layout and channel assignments, and tests direct from falcon and xlights we successful but FPP via the PI was not behaving and 

I've tried to include configuration detail from the F48 and FPP highlighting the affected output.  

Again, i realize it's a very bizarre relationship but can reproduce the issue by simply switching to USA.

FPP_ProblematicSetting
![FPP_ProblemSetting](https://user-images.githubusercontent.com/46975715/102149145-642f2e80-3e33-11eb-9978-99cdbdddd2e1.jpg)

F48_Status_LocaleSetToUSA
![F48_Status_LocaleSetToUSA](https://user-images.githubusercontent.com/46975715/102148874-e23f0580-3e32-11eb-98d9-a9cc7b76fdfe.jpg)

F48_Status_LocaleSetToGlobal
![F48_Status_LocaleSetToGlobal](https://user-images.githubusercontent.com/46975715/102148905-ed923100-3e32-11eb-9023-1e0226657708.jpg)

F48_PortConfig_AffectedOutputs
![F48_PortConfig_AffectedOutputs](https://user-images.githubusercontent.com/46975715/102148927-fedb3d80-3e32-11eb-912b-6e831b975acb.jpg)

F48_E131_AffectedUniverse
![F48_E131_AffectedUniverse](https://user-images.githubusercontent.com/46975715/102148949-0ac6ff80-3e33-11eb-9947-2553eef12bce.jpg)

FPP_E131_AffectedConfig
![FPP_E131_AffectedConfig](https://user-images.githubusercontent.com/46975715/102148969-14e8fe00-3e33-11eb-8f20-4ee62596d22f.jpg)

FPP_PixelOverlay_AffectedModels
![FPP_PixelOverlay_AffectedModels](https://user-images.githubusercontent.com/46975715/102148990-20d4c000-3e33-11eb-9405-6d451a9d2500.jpg)

Logs:
[FPP_Logs_20201214.2314.zip](https://github.com/FalconChristmas/fpp/files/5692179/FPP_Logs_20201214.2314.zip)

**Comment by cpinkham** [comment]
Are you also changing the Dedup setting on these universes or some other setting?  Your packet counts per universe vary wildly on one screenshot but are almost exactly in sync in the other screen shot.

**Comment by darylc** [comment]
What happens if you untick the Monitor in the FPP UDP outputs screen?  I saw a case to this recently where some universes would get no data in some scenarios when this was ticked.

**Comment by darylc** [comment]
@jimtxas

**Comment by jimtxas** [comment]
Apologies for the delay.  Thought i had subscribed to notifications but didn't receive until you tagged...

@cpinkham  , I'm not making any other changes.  I agree, that's odd so I verified again by simply checking only the locale box to USA and much to my dismay the problem is not reproducing today.  Absolutely no other changes nor even sequence uploads/changes since i opened this ticket and worked through the issue believing i had a smoking gun - very frustrating. 

Now a bit more information though - i had a scenario just last night where i noticed 2-3 strings on the megatree were not lighting.  Based on wind i believed this a bad cable that i was going to have to replace but ran test pattern at controller and they lit up.  Went to FPP test and same ones were out and the packet count was showing off on that controller (different one - F16v3 in that case) and entirely random universes not functioning from FPP.  I rebooted from FPP (not just an FPPD restart) and it didn't resolve the issue.  In desperation and frustration i did a full power cycle on the PI and all of them came back and started receiving data.

Based on my inability to reproduce now at this moment I'm no longer certain that locale setting had any impact and perhaps this is a PI/hardware issue?  I am certain in that moment when i opened the ticket that the locale setting was clearly affecting that behavior but perhaps that coupled somehow with how the pi was behaving.  If and when it happens again is there any sort of data that i should collect when it's misbehaving that could be useful?  I've got a couple of other pi's laying around i can load up and put in play but would kind of like to see it misbehave and then try @darylc's idea with the monitor option.  Maybe it's something there but since the issue will not reproduce at the moment cannot clearly answer that question.

Apologies for the goose chase on the locale settings, but again in that moment it was clearly affecting behavior but likely just a anectodical influence.  I can confirm in all my troubleshooting i was not doing hard power resets which likely would have solved the problem in that moment but was rather only using the reboot or restart FPPD options in FPP.
